templates/base.html

```diff
@@ -83,14 +58,16 @@
             </li>
           </ul>
           {% if user.is_authenticated %}
+          <a id class="nav-link me-sm-2" href="/accounts/edit_profile">Edit Profile</a>
```

Comment:
this url was not regitered anywhere and we get 404 when pressing edit_profile
![image](https://user-images.githubusercontent.com/34008861/170834854-2bbe55b0-7c8b-4143-a049-50b32c095c37.png)

Reply:
Fixed